src/Generator/Generators/CSharp/CSharpSources.cs

```diff
@@ -364,6 +363,55 @@ public override bool VisitClassDecl(Class @class)
 
                 if (@class.IsDynamic)
                     GenerateVTable(@class);
+            } else
```

Comment:
If we do, they will be to specific pieces in the large block. So we'd better off just with one block for the time being.